Automattic/woocommerce-payments
# fix: onboarding task on account task list disabled

Fixes #

#### Changes proposed in this Pull Request









The [onboarding task page](http://localhost:8082/wp-admin/admin.php?page=wc-admin&task=woocommerce-payments--additional-payment-methods) is blank when `_wcpay_feature_account_overview_task_list` is disabled.
With these changes, the task will be visible again.

#### Testing instructions





- Remove `_wcpay_feature_account_overview_task_list` from your DB
- Ensure "Enable UPE settings changes" is enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-admin&task=woocommerce-payments--additional-payment-methods
- Task is displayed
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Foverview
- Task list is not displayed
- Add `_wcpay_feature_account_overview_task_list` in your `wp_options`
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Foverview
- Task list is displayed

![Screen Shot 2021-07-20 at 2 47 15 PM](https://user-images.githubusercontent.com/273592/126386285-7150c53a-9c07-42fb-ac99-d8b9300b59e6.png)


-------------------
